api/core/model_runtime/model_providers/openllm/llm/openllm_generate.py

```diff
@@ -41,9 +41,9 @@ def generate(
         model_name: str,
         stream: bool,
         model_parameters: dict[str, Any],
-        stop: list[str],
-        prompt_messages: list[OpenLLMGenerateMessage],
-        user: str,
+        stop: list[str] | None = None,
+        prompt_messages: list[OpenLLMGenerateMessage] = [],
```

Comment:
Please avoid using reference types as default parameter values, as this can lead to potential memory leak errors.

Reply:
yes, but I think we need to change the order, for default value, python needs them in the last of the function args.
I will use Options type here for `stop` what do you think

Reply:
I have a simpler proposal: we can enforce users to pass arguments by name by adding a `*` after the `self` parameter. This approach will make the code more readable.

```python
    def generate(
        self,
        *,
        server_url: str,
        ...
```